### PR TITLE
Chore: tactic alive_auto

### PR DESCRIPTION
**Pull request by AnotherAlexHere**
Changed all 
```lean
sorry
```
 to 
```lean
try alive_auto
all_goals sorry
```

### CODE REVIEW

File: SSA/Projects/InstCombine/HackersDelight.lean [hunk 1]
```diff
@@ -19,11 +23,14 @@ theorem not_eq_neg_sub_one :
 
 theorem not_and_eq_not_or_not :
     ~~~ (x &&& y) = ~~~ x ||| ~~~ y := by
-  sorry
+  try alive_auto
+  all_goals try alive_auto
+  all_goals sorry
```
Comment (AtticusKuhn) [inline SSA/Projects/InstCombine/HackersDelight.lean]:
```suggestion
  try alive_auto
  all_goals sorry
```

**Review by AtticusKuhn — APPROVED** [review]
LGTM

**Comment by AtticusKuhn** [comment]
How many theorems does `alive_auto` solve?

**Review by Equilibris — COMMENTED** [review]
LGTM, but at the same time i think a discussion should be opened on whether this is a good idea in general when some of the theorems don't solve by the tactics as the complexity is $\mathcal O (n^2)$

**Review by Equilibris — APPROVED** [review]

**Comment by AtticusKuhn** [comment]
> LGTM, but at the same time i think a discussion should be opened on whether this is a good idea in general when some of the theorems don't solve by the tactics as the complexity is O ( n 2 )

This is a good observation. Maybe we could break up this file into smaller files to speed this up?

**Pull request merged**